src/CommunityToolkit.Maui.DeviceTests/CommunityToolkit.Maui.DeviceTests/Tests/BehaviorTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Behaviors;
+using Xunit;
+
+namespace CommunityToolkit.Maui.DeviceTests.Tests;
+public class BehaviorTest
+{
+	[Fact]
+	public void SuccessfulTest()
+	{
+		Assert.True(true);
+	}
+
+	[Fact]
```

Comment:
Should it be a UIFact?

Reply:
No, UIFact is just when need to test UI-related stuff. For example, to check if the expander is expanded after some action.